lookahead_distance: 3
axle_length: 1.5
Kpp: 0.90
Kp: 0.09
Ki: 0.01
weight_current: 0.9
filter_length: 10